modules/main/src/main/env/service/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
@felixgao Mind moving the modules/main/src/main/env/service/entrypoint.sh change to https://github.com/intuit/wasabi/pull/31 then so it's in one place?